ovos_ww_plugin_vosk/__init__.py

```
@@ -373,3 +380,105 @@
+        url (str): URL of file to download
+        file (Union[str, io, None]): One of the following:
+             - Filename of output file
+             - File opened in binary write mode
+             - None: Return raw bytes instead
+
+    Returns:
+        Union[bytes, None]: Bytes of file if file is None
+    """
+
+    if isinstance(file, str):
+        file = open(file, 'wb')
```

Ruff

Use a context manager for opening files

(SIM115)